rpa-mapper/src/test/resources/defence/defence_response_company_rpa_case.json

```diff
@@ -0,0 +1,27 @@
+{
+  "caseNumber": "000CM001",
+  "defenceSubmittedOn": "26-Apr-2018",
+  "defenceResponse": "I dispute all the claim",
```

Comment:
Agreed with business. Would like to leave it like that. We can relook into it with new stories we have work on full admission stories

Reply:
Changing it now is much easier then doing this later. That is the reason I am trying to achieve good initial structure because every later refactoring will cost us much more.

If communication is the problem reach to RPA directly.

Reply:
Changed it to enum name.